Setting-Up-Twitch.md

```diff
@@ -0,0 +1,25 @@
+### What is Twitch?
+
+Twitch is a live streaming video platform that was founded in June 2011. More than 1.7 million people broadcast on Twitch each month for 100 million unique visitors each month. Click [here](https://www.twitch.tv/p/about) to find out more about Twitch and it's statistics.
```

Comment:
...people broadcast on Twitch each month ~~for~~ **with** 100 million unique visitors. ~~each month~~